frontend/src/utils/index.js

```diff
@@ -685,6 +685,11 @@ export function normalizeVersion (version) {
   }
 }
 
+export function getResourceRefName (resources, resourceName) {
```

Comment:
I think this should not be something for general purpose utils. From my point of view it would be better nd more readable to reuse the list of resources and duplicate accessing the `resorceRef.name`.

Reply:
removed utils function